src/components/side-navigation/SideNavigation.tsx

```diff
@@ -1,11 +1,17 @@
 import style from './SideNavigation.module.sass';
 import NavigationHeader from './NavigationHeader';
 import NavigationItem from './NavigationItem';
-
+import { SideNavigationContext } from '../../store/SideNavigationContext';
+import { useContext } from 'preact/hooks';
 /**
  * The side navigation bar of the website.
  */
 export default function SideNavigation() {
+  const sideNavContext = useContext(SideNavigationContext);
+  const isOpen = sideNavContext.isSideNavigationOpen;
+  if(!isOpen){
```

Comment:
Can be simplified by `return sideNavContext.isSideNavigationOpen ? (<aside ...>) : null`